src/components/structures/SpaceRoomView.tsx

```diff
@@ -897,10 +897,12 @@ export default class SpaceRoomView extends React.PureComponent<IProps, IState> {
                     title={_t("What are some things you want to discuss in %(spaceName)s?", {
                         spaceName: this.props.justCreatedOpts?.createOpts?.name || this.props.space.name,
                     })}
-                    description={
-                        _t("Let's create a room for each of them.") + "\n" +
-                        _t("You can add more later too, including already existing ones.")
-                    }
+                    description={_t(
+                        "Let's create a room for each of them.<br />" +
+                        "You can add more later too, including already existing ones.",
+                        {},
+                        { 'br': <br /> },
+                    )}
```

Comment:
How about something more like 

```typescript
description={<>
  { _t("Let's create a room for each of them.") }
  <br />
  { _t("You can add more later too, including already existing ones.") }
</>}
```

Reply:
Thank you. Obviously I did not know what I was doing :sweat_smile: